cms/navigation/models.py

```diff
@@ -0,0 +1,107 @@
+from typing import TYPE_CHECKING, ClassVar
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from wagtail.admin.panels import FieldPanel
+from wagtail.blocks import CharBlock, ListBlock, PageChooserBlock, StructBlock
+from wagtail.contrib.settings.models import register_setting
+from wagtail.models import DraftStateMixin, PreviewableMixin, RevisionMixin
+
+from cms.core.blocks.base import LinkBlock
+from cms.core.fields import StreamField
+from cms.core.models import BaseSiteSetting
+
+MAIN_MENU_MAX_NUM_HIGHLIGHTS = 3
+MAIN_MENU_MAX_NUM_COLUMNS = 3
+MAIN_MENU_MAX_NUM_SECTIONS = 3
+MAIN_MENU_MAX_NUM_TOPIC_LINKS = 15
```

Comment:
To be honest, I am don't think these add much value.

Reply:
I thought this would be easy to follow and read (even though I know these numbers may not change often). I'm not too picky about this, so I'm happy to remove it.